velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -2858,6 +2858,52 @@ TEST_F(DateTimeFunctionsTest, dateParse) {
       dateParse("116", "%y+"), "Invalid format: \"116\" is malformed at \"6\"");
 }
 
+// The test for the alias now() is integrated into this test.
+TEST_F(DateTimeFunctionsTest, currentTimestamp) {
+  const auto evaluateCurrentTimestamp =
+      [&](const std::string& functionName,
+          int64_t expectedTimestamp,
+          const std::string& expectedTimeZone) {
+        auto emptyRowVector = makeRowVector(ROW({}), 1);
+
+        auto callExpr = std::make_shared<const core::CallTypedExpr>(
+            TIMESTAMP_WITH_TIME_ZONE(),
+            std::vector<core::TypedExprPtr>{},
+            functionName);
+
+        auto actual = evaluate(callExpr, emptyRowVector);
+
+        auto expected = makeTimestampWithTimeZoneVector(
+            expectedTimestamp, expectedTimeZone.c_str());
+
+        ASSERT_EQ(expected->size(), actual->size());
```

Comment:
So expected is a RowVectorPtr with 2 vectors as children. expected->childAt(0).. should give you the VectorPtr for the timestamp. Similarly actual->childAt(0) should give the VectorPtr for the timestamp.

My thinking is for the timestamps we should compare actual->childAt(0)->compare(expected->childAt(0), 0, 0) and for timezone we should compare actual->childAt(1)->compare(expected->childAt(1), 0, 0)